.github/workflows/lra-test.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: LRA test
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+  pull_request:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+
+jobs:
+  lra-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Narayana - build
+      run: ./build.sh clean install -Pcommunity -DskipTests -Didlj-enabled=true
+    - name: LRA - test (tck/basic)
+      run: |
+        cd rts/lra/lra-test
```

Comment:
It seems this is not something in the current narayana.sh? https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh

Reply:
There isn't a possibility to run the full test suite. The issue is with the tests in lra-coordinator-jar module because they require full WFLY server which is not downloaded included during the builds. As mentioned above this is just a quick setup mainly for TCK (but also includes basic test suite) so we can catch timing issues in the Hudson CI. We will build on top of this if necessary later. I am not sure if Narayana team would want to migrate all testing to GH actions.

Reply:
I see, I am not sure if just running the TCK would give that much information. It would be better if the whole of the LRA testing could take place to replicate the existing LRA `PROFILE` in narayana.sh if possible